app/models/candidate.rb

```diff
@@ -1,3 +1,12 @@
 class Candidate < ApplicationRecord
   belongs_to :user
+  has_many :resume, dependent: :destroy
+  has_many :applies, dependent: :destroy
+
+  validates :small_desc,length: { minimum: 10 }
```

Comment:
validates :small_desc, length: { in: 30..240 }

Reply:
ok!